Makefile

```diff
@@ -0,0 +1,15 @@
+all: test
```

Comment:
> I am 👎 on adding makefiles, they are not cross platform and don't add much value compared to just npm run-script

Wrong. They add an enormous amount of value for users of tools cross language. The point of `make` is that -- no matter who wrote something and what they use daily -- a `Makefile` is a language-agnostic tool, so anyone can jump in and understand what to do. I'm not suggesting you use `make` to do the heavy lifting. I'm suggesting the use of `Makefiles` in places like Orbit, or orbit-db, where people will download the repos and want to do something, but (a) don't know npm well. (b) don't know how to invoke things. (c) dont know _which_ things to invoke. `make` is a _standard_ way to do it.

also, i dont know what the current distribution of people with Windows, unix tooling, and no make program. It used to be quite small. maybe someone should compile `make` with emscripten and publish it to npm.